AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -18,6 +18,7 @@ @implementation ASLayoutSpec
 
 // Dynamic properties for ASLayoutElements
 @dynamic layoutElementType;
+@synthesize debugName = _debugName;
```

Comment:
I think you probably don't need this line, because you implemented the getter and setter method.  See if it compiles, since you do need it to create the _debugName instance variable, but I think it will do that from the @property alone.